tools/SourceKit/tools/sourcekitd-test/TestOptions.cpp

```diff
@@ -290,5 +294,11 @@ bool TestOptions::parseArgs(llvm::ArrayRef<const char *> Args) {
     return true;
   }
 
+  if (Timings) {
+    PrintRequest = false;
+    PrintResponseAsJSON = false;
+    PrintRawResponse = false;
+  }
```

Comment:
Booo.  Okay, my comment stands for the other printing options, and we should probably add a `-print-request`, but that's a separate issue.